src/System.Security.Cryptography.X509Certificates/tests/X509FilesystemTests.Unix.cs

```diff
@@ -15,8 +15,12 @@ namespace System.Security.Cryptography.X509Certificates.Tests
     [Collection("X509Filesystem")]
     public static class X509FilesystemTests
     {
-        // #9293: Our Fedora23 and Ubuntu1610 CI machines use NTFS for "tmphome", which causes our filesystem permissions checks to fail.
-        private static bool IsReliableInCI { get; } = !PlatformDetection.IsFedora23 && !PlatformDetection.IsUbuntu1610;
+        // #9293: Our Fedora and Ubuntu CI machines use NTFS for "tmphome", which causes our filesystem permissions checks to fail.
+        private static bool IsReliableInCI { get; } =
```

Comment:
It's probably better to make the one in ChainTests be internal, and have this file use

```C#
private static bool IsReliableInCI = ChainTests.IsReliableInCI;
```

so the list doesn't need to be duplicated.  (it needs to be that direction so that the Windows build compiles)